src/main/resources/jenkins/plugins/publish_over_ftp/BapFtpPublisher/config.jelly

```diff
@@ -66,14 +66,14 @@
         <j:scope>
             <j:set var="descriptor" value="${descriptor.transferDescriptor}"/>
             <f:repeatable items="${instance.transfers}" var="instance" name="transfers" minimum="1" header="${m.transfers_dragAndDropLabel()}" add="${m.addTransfer()}">
-                <table width="100%" padding="0">
+                <p:blockWrapper>
```

Comment:
isn't there more files that need changing? see https://github.com/jenkinsci/publish-over-ftp-plugin/search?q=table

Reply:
Ups! You are totally right